wp-content/themes/engage/templates/single.twig

```diff
@@ -26,7 +27,7 @@
                         </iframe>
                       </div>
                     </div>
-                  {% elseif hasFeaturedImage %}
+                  {% elseif hasFeaturedImage and displayFeaturedImage == 'display' %}
```

Comment:
Since we already have a default of showing the featured image, this release would require an update of all posts across thee board to set this value to `display`. Rather than do that (since by default all images would be hidden), let's default to showing it and rename this to `post.display_featured_image` and have a boolean true/false with a default to `true`. That way, it can be hidden via a checkbox in the admin ("Hide featured image on post: `[checkbox]`".

Let me know your thoughts or if you have questions!